src/components/AvatarWithImagePicker.tsx

```diff
@@ -1,181 +1,152 @@
-    originalFileName: PropTypes.string,
+    originalFileName?: string;
 
     /** Whether navigation is focused */
-    isFocused: PropTypes.bool.isRequired,
+    isFocused: boolean;
 
     /** Style applied to the avatar */
-    avatarStyle: stylePropTypes.isRequired,
+    avatarStyle: StyleProp<ViewStyle>;
 
     /** Indicates if picker feature should be disabled */
-    disabled: PropTypes.bool,
+    disabled?: boolean;
 
     /** Executed once click on view photo option */
-    onViewPhotoPress: PropTypes.func,
-
-    /** Where the popover should be positioned relative to the anchor points. */
-    anchorAlignment: PropTypes.shape({
-        horizontal: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL)),
-        vertical: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_VERTICAL)),
-    }),
-};
-
-const defaultProps = {
-    source: '',
-    onImageSelected: () => {},
-    onImageRemoved: () => {},
-    style: [],
-    disabledStyle: [],
-    DefaultAvatar: () => {},
-    isUsingDefaultAvatar: false,
-    size: CONST.AVATAR_SIZE.DEFAULT,
-    fallbackIcon: Expensicons.FallbackAvatar,
-    type: CONST.ICON_TYPE_AVATAR,
-    editorMaskImage: undefined,
-    errorRowStyles: [],
-    onErrorClose: () => {},
-    pendingAction: null,
-    errors: null,
-    headerTitle: '',
-    previewSource: '',
-    originalFileName: '',
-    disabled: false,
-    onViewPhotoPress: undefined,
-    anchorAlignment: {
-        horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
-        vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.TOP,
-    },
+    onViewPhotoPress?: () => void;
 };
 
 function AvatarWithImagePicker({
     isFocused,
-    DefaultAvatar,
+    DefaultAvatar = () => null,
     style,
     disabledStyle,
     pendingAction,
     errors,
     errorRowStyles,
-    onErrorClose,
-    source,
-    fallbackIcon,
-    size,
-    type,
-    headerTitle,
-    previewSource,
-    originalFileName,
-    isUsingDefaultAvatar,
-    onImageRemoved,
-    onImageSelected,
+    onErrorClose = () => {},
+    source = '',
+    fallbackIcon = Expensicons.FallbackAvatar,
+    size = CONST.AVATAR_SIZE.DEFAULT,
+    type = CONST.ICON_TYPE_AVATAR,
+    headerTitle = '',
+    previewSource = '',
+    originalFileName = '',
+    isUsingDefaultAvatar = false,
+    onImageSelected = () => {},
+    onImageRemoved = () => {},
     editorMaskImage,
     avatarStyle,
-    disabled,
+    disabled = false,
     onViewPhotoPress,
-}) {
+}: AvatarWithImagePickerProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const {windowWidth} = useWindowDimensions();
     const [popoverPosition, setPopoverPosition] = useState({horizontal: 0, vertical: 0});
     const [isMenuVisible, setIsMenuVisible] = useState(false);
-    const [errorData, setErrorData] = useState({
-        validationError: null,
-        phraseParam: {},
-    });
+    const [errorData, setErrorData] = useState<ErrorData>({validationError: '', phraseParam: {}});
```

Comment:
Let's turn it back to null, this way we can remove `''` from ErrorData typing
```suggestion
    const [errorData, setErrorData] = useState<ErrorData>({validationError: null, phraseParam: {}});
```